plugins/hello_world_plugin/.ci/ci_matrix.yaml

```diff
@@ -0,0 +1,54 @@
+---
+job: ufm-hello-world-plugin
+
+registry_host: harbor.mellanox.com
+registry_path: /swx-storage/ci-demo
+registry_auth: swx-storage
+
+env:
+  plugin_dir: hello_world_plugin
+  plugin_name: ufm-plugin-hello-world
+  DOCKER_CLI_EXPERIMENTAL: enabled
+
+kubernetes:
+  cloud: swx-k8s-spray
+
+volumes:
+  - {mountPath: /var/run/docker.sock, hostPath: /var/run/docker.sock}
+  - {mountPath: /auto/UFM, hostPath: /auto/UFM }
+
+
+runs_on_dockers:
+   - {file: '.ci/Dockerfile', arch: 'x86_64', name: 'plugin_worker', tag: 'latest'}
+
+
+steps:
+  - name: Build Plugin
+    containerSelector: "{name: 'plugin_worker'}"
+    run: |
+      cd plugins/$plugin_dir/build
+      bash -x ./docker_build.sh latest /
+      ls -l /
+      cp /ufm-plugin* /auto/UFM/tmp/${JOB_NAME}/${BUILD_ID}/
```

Comment:
not sure that all plugins have this naming convention

Reply:
All of them have this prefix 
![image](https://github.com/Mellanox/ufm_sdk_3.0/assets/15429511/b99b3f99-da3c-4158-a291-d96b0baa3d9e)